packages/flutter_tools/test/general.shard/macos/cocoapods_test.dart

```diff
@@ -181,13 +181,13 @@ void main() {
 
     testWithoutContext('detects at recommended version', () async {
       pretendPodIsInstalled();
-      pretendPodVersionIs('1.8.0');
+      pretendPodVersionIs('1.9.0');
       expect(await cocoaPodsUnderTest.evaluateCocoaPodsInstallation, CocoaPodsStatus.recommended);
     });
 
     testWithoutContext('detects above recommended version', () async {
       pretendPodIsInstalled();
-      pretendPodVersionIs('1.8.1');
+      pretendPodVersionIs('1.9.1');
```

Comment:
not a blocker, but should we be warning if the .z is above the recommended version?

Reply:
Hm, what do you mean? This test shows it returning `CocoaPodsStatus.recommended` which doesn't translate to a warning.

Reply:
ahh, sorry, you're right.